docs/03-tutorials/api-exposure/apix-04-own-domain.md

```diff
@@ -0,0 +1,200 @@
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSProvider
+   metadata:
+     name: dns-provider
+     namespace: NAMESPACE_NAME # Edit this value
+     annotations:
+       dns.gardener.cloud/class: garden
+   spec:
+     type: your-provider-dns-type # Edit this value
+     secretRef:
+       name: secret-name # Edit this value
+     domains:
+       include:
+         - mydomain.com # The main domain. Edit this value
+   EOF  
+   ```
+
+- Create a DNSEntry CR. See the following example and modify values for the **metadata.namespace**, **metadata.spec.dnsName**, and **metadata.spec.targets.IP** parameters. Optionally, you can also change the value of **metadata.spec.ttl**. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: dns.gardener.cloud/v1alpha1
+   kind: DNSEntry
+   metadata:
+     name: dns-entry
+     namespace: NAMESPACE_NAME # Edit this value
+     annotations:
+       dns.gardener.cloud/class: garden
+   spec:
+     dnsName: "my.sub1.mydomain.com" # Your subdomain. Edit this value
+     ttl: 600
+     targets:
+       - 1.2.3.4 # IP address. Edit this value
+   EOF
+   ```
+
+    >**NOTE:** To check the Ingress Gateway IP, run:
+    >```
+    >kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+    >```
+
+   >**NOTE:** You can create many DNSEntry CRs for one DNSProvider, depending on the number of subdomains you want to use.
+
+5. Create an Issuer CR. See the following example and modify values of the **metadata.spec.acme.email**, **metadata.spec.domains.include**, and **metadata.spec.domains.exclude** parameters. As the values for the **metadata.spec.domains.include** parameter, use the main domain, the subdomain from the **spec.dnsName** parameter of the DNSEntry CR, and a wildcard DNS record. Run:
+
+   ```bash
+   cat <<EOF | kubectl apply -f -
+   apiVersion: cert.gardener.cloud/v1alpha1
+   kind: Issuer
+   metadata:
+     name: letsencrypt-staging
+     namespace: default
+   spec:
+     acme:
+       server: https://acme-staging-v02.api.letsencrypt.org/directory
+       email: YOUR_EMAIL_ADDRESS # Edit this value
+       autoRegistration: true
+       # Name of the Secret used to store the ACME account private key
+       # If doesn't exist, a new one is created
+       privateKeySecretRef:
+         name: letsencrypt-staging-secret
+         namespace: default
```

Comment:
Minor issue, but maybe the target tutorial namespace (the one with other objects) should be used?

Reply:
Let's leave it as it is.